tests/at_functional_test/test/delete_verb_test.dart

```diff
@@ -152,6 +153,35 @@ void main() {
         (!response.contains('Invalid syntax')) && (!response.contains('null')));
   }, timeout: Timeout(Duration(seconds: 50)));
 
+  test('delete verb for an protected key - signing_publickey', () async {
+    ///DELETE VERB
+    await socket_writer(
+        socketFirstAtsign!, 'delete:public:signing_publickey$firstAtsign');
+    var response = await read();
+    print('delete verb response : $response');
+    // the error is an expected behaviour
+    assert((response.contains(
+            'UnAuthorized client in request : Cannot delete protected key')) &&
+        (response.contains('error')));
+
+    ///SCAN VERB
+    await socket_writer(socketFirstAtsign!, 'scan');
+    response = await read();
+    print('scan verb response is :$response');
+    // ensure that the signing_publickey is not deleted
+    assert(response.contains('signing_publickey'));
+  });
+
+  test('delete verb for an protected key - pkam_publickey', () async {
+    ///DELETE VERB
+    await socket_writer(
+        socketFirstAtsign!, 'delete:privatekey:at_pkam_publickey');
+    var response = await read();
+    print('delete verb response : $response');
+    // the error is an expected behaviour
+    assert((response.contains('Invalid syntax')));
+  });
+
```

Comment:
Please add explicit tests for the other two core protected keys (public encryption key, signing private key)

Reply:
These tests have been added through 6fcdede737deb85ee00f32961f10fffa2d0c9b1a